### PR TITLE
Split AdGuard Annoyances filter

### PR DESCRIPTION
**Pull request by zloyden**
Do not merge this pull request until this one get approve with translated locales and another related pull request in AdGuardFilters is merged by me.
Related pull request and tasks from other repositories
https://github.com/AdguardTeam/FiltersRegistry/issues/632
- [x] https://github.com/AdguardTeam/AdguardFilters/pull/123739
- [x] https://github.com/AdguardTeam/AdguardFilters/issues/123836
- [x] https://github.com/AdguardTeam/AdguardKnowledgeBase/issues/1093

### CODE REVIEW

File: filters/filter_20_Annoyances_MobileApp/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 20,
+  "name": "AdGuard MobileApp filter",
```
Comment (ameshkov) [inline filters/filter_20_Annoyances_MobileApp/metadata.json:3]:
```suggestion
  "name": "AdGuard Mobile App Banners filter",
```

File: filters/filter_21_Annoyances_Other/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 21,
+  "name": "AdGuard Other filter",
```
Comment (ameshkov) [inline filters/filter_21_Annoyances_Other/metadata.json:3]: I am not entirely sure about the proposed name though.

```suggestion
  "name": "AdGuard Other Annoyances filter",
```

**Review by ameshkov — CHANGES_REQUESTED** [review]
Please ask the content team to review the names & descriptions

File: filters/filter_21_Annoyances_Other/metadata.json (hunk 2 above)
Comment (zloyden) [reply filters/filter_21_Annoyances_Other/metadata.json:3]: It's a draft name, that can be changed.

**Comment by zloyden** [comment]
@el-termikael please review the names & descriptions.

File: filters/filter_14_Annoyances/metadata.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 14,
   "name": "AdGuard Annoyances filter",
-  "description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups.",
+  "description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard MoblieApp, AdGuard Other, AdGuard Widgets.",
```
Comment (el-termikael) [inline filters/filter_14_Annoyances/metadata.json:4]: AdGuard MoblieApp?

File: filters/filter_21_Annoyances_Other/metadata.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 21,
+  "name": "AdGuard Other Annoyances filter",
+  "description": "Blocks irritating elements on web pages that do not fall under the popular annoyances categories.",
```
Comment (el-termikael) [inline filters/filter_21_Annoyances_Other/metadata.json:4]: the popular annoyances categories.
—>
the popular categories of annoyances.

**Review by el-termikael — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

File: filters/filter_19_Annoyances_Popups/metadata.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 19,
+  "name": "AdGuard Popups filter",
+  "description": "Blocks in-page pop-ups on web pages. Already included in AdGuard Annoyances filter.",
```
Comment (Sergey-Lyapin) [inline filters/filter_19_Annoyances_Popups/metadata.json:4]: pop-ups and inline subscriptions?

Comment (zloyden) [reply filters/filter_19_Annoyances_Popups/metadata.json:4]: I'm not sure about adding this. Yes, we have inline subscriptions, but also popups.

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Comment by zloyden** [comment]
Waiting for translations.

**Review by Alex-302 — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

File: filters/filter_18_Annoyances_Cookies/metadata.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  "name": "AdGuard Cookies filter",
+  "description": "Blocks cookie notices on web pages. Already included in AdGuard Annoyances filter.",
+  "timeAdded": 1657180419636,
+  "homepage": "https://kb.adguard.com/general/adguard-ad-filters",
+  "expires": "4 days",
+  "displayNumber": 2,
+  "groupId": 4,
+  "subscriptionUrl": "https://filters.adtidy.org/extension/chromium/filters/18.txt",
+  "tags": [
+    "purpose:cookies",
```
Comment (Alex-302) [inline filters/filter_18_Annoyances_Cookies/metadata.json:12]: Maybe `purpose:annoyances`?

File: filters/filter_14_Annoyances/metadata.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 14,
   "name": "AdGuard Annoyances filter",
-  "description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets.",
+  "description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets filters.",
```
Comment (el-termikael) [inline filters/filter_14_Annoyances/metadata.json:4]: Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances and AdGuard Widgets.

File: locales/en/filters.json [hunk 8]
```diff
@@ -53,7 +53,7 @@
 	},
 	{
 		"filter.14.name": "AdGuard Annoyances filter",
-		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets."
+		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets filters."
```
Comment (el-termikael) [inline locales/en/filters.json:56]: Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances and AdGuard Widgets.

File: filters/filter_14_Annoyances/metadata.json (hunk 7 above)
Comment (el-termikael) [reply filters/filter_14_Annoyances/metadata.json:4]: ну или Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances and AdGuard Widgets filters.

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: filters/filter_18_Annoyances_Cookies/metadata.json (hunk 6 above)
Comment (zloyden) [reply filters/filter_18_Annoyances_Cookies/metadata.json:12]: Why so? It's only about the cookies.

File: filters/filter_14_Annoyances/metadata.json [hunk 9]
```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 14,
   "name": "AdGuard Annoyances filter",
-  "description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets filters.",
+  "description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets.",
```
Comment (el-termikael) [inline filters/filter_14_Annoyances/metadata.json:4]: Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances and AdGuard Widgets — лучше с союзом

File: filters/filter_20_Annoyances_MobileApp/metadata.json [hunk 10]
```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 20,
   "name": "AdGuard Mobile App Banners filter",
-  "description": "Blocks banners that promote the mobile apps of websites. Already included in AdGuard Annoyances filter.",
+  "description": "Blocks banners that promote irritating mobile apps of websites. Already included in AdGuard Annoyances filter.",
```
Comment (el-termikael) [inline filters/filter_20_Annoyances_MobileApp/metadata.json:4]: эм, наверное, баннеры раздражают, а не приложения?)
Blocks irritating banners that promote mobile apps of websites

File: locales/en/filters.json [hunk 11]
```diff
@@ -53,7 +53,7 @@
 	},
 	{
 		"filter.14.name": "AdGuard Annoyances filter",
-		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. The filter contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets filters."
+		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances, AdGuard Widgets."
```
Comment (el-termikael) [inline locales/en/filters.json:56]: и тут and не хватает

File: locales/en/filters.json [hunk 12]
```diff
@@ -77,7 +77,7 @@
 	},
 	{
 		"filter.20.name": "AdGuard Mobile App Banners filter",
-		"filter.20.description": "Blocks banners that promote the mobile apps of websites. Already included in AdGuard Annoyances filter."
+		"filter.20.description": "Blocks banners that promote irritating mobile apps of websites. Already included in AdGuard Annoyances filter."
```
Comment (el-termikael) [inline locales/en/filters.json:80]: irritating banners

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: filters/filter_20_Annoyances_MobileApp/metadata.json (hunk 10 above)
Comment (zloyden) [reply filters/filter_20_Annoyances_MobileApp/metadata.json:4]: Это хорошо подмечено, не туда улетело)

**Review by Alex-302 — APPROVED** [review]

**Review by el-termikael — APPROVED** [review]

File: locales/en/filters.json [hunk 13]
```diff
@@ -53,7 +53,7 @@
 	},
 	{
 		"filter.14.name": "AdGuard Annoyances filter",
-		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups."
+		"filter.14.description": "Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups. Contains AdGuard Cookies, AdGuard Popups, AdGuard Mobile App Banners, AdGuard Other Annoyances and AdGuard Widgets."
```
Comment (ameshkov) [inline locales/en/filters.json:56]: Too many AdGuards there eh?

Contains Cookie Notices, Popups, Mobile App Banners, Widgets and Other Annoyances filters.

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: filters/filter_20_Annoyances_MobileApp/metadata.json [hunk 14]
```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 20,
+  "name": "AdGuard Mobile App Banners filter",
+  "description": "Blocks irritating banners that promote mobile apps of websites. Already included in AdGuard Annoyances filter.",
```
[the remark this answers is not in the document]
Comment (zloyden) [reply filters/filter_20_Annoyances_MobileApp/metadata.json:4]: Not to confuse the user, but if it is ok to leave only one place for that (AdGuard Annoyances), well, I can change it.

File: locales/en/filters.json (hunk 13 above)
Comment (zloyden) [reply locales/en/filters.json:56]: Yes, but it also reflects their names. If we can leave it the way you suggested, fine)

Comment (ameshkov) [reply locales/en/filters.json]: I think we can

Comment (zloyden) [reply locales/en/filters.json]: I did it, thanks.

File: filters/filter_19_Annoyances_Popups/metadata.json (hunk 5 above)
Comment (ameshkov) [inline filters/filter_19_Annoyances_Popups/metadata.json:4]: Can we make explain a bit more? What exactly are these pop-ups?

File: filters/filter_20_Annoyances_MobileApp/metadata.json [hunk 15]
```diff
@@ -0,0 +1,17 @@
...
+  "timeAdded": 1657180419636,
+  "homepage": "https://kb.adguard.com/general/adguard-ad-filters",
+  "expires": "4 days",
+  "displayNumber": 2,
+  "groupId": 4,
+  "subscriptionUrl": "https://filters.adtidy.org/extension/chromium/filters/20.txt",
+  "tags": [
+    "purpose:annoyances",
```
Comment (ameshkov) [inline filters/filter_20_Annoyances_MobileApp/metadata.json:12]: purpose:mobile maybe?

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: filters/filter_20_Annoyances_MobileApp/metadata.json (hunk 15 above)
Comment (slavaleleka) [reply filters/filter_20_Annoyances_MobileApp/metadata.json:12]: we have `platform:mobile`

Comment (zloyden) [reply filters/filter_20_Annoyances_MobileApp/metadata.json:12]: Hm, we don't have it, but I can add it in this pr.

Comment (slavaleleka) [reply filters/filter_20_Annoyances_MobileApp/metadata.json:12]: but it is not only about mobile, sometimes mobile app banners appears in desktop versions of websites and we block them too (or we used to block them)

Comment (ameshkov) [reply filters/filter_20_Annoyances_MobileApp/metadata.json:12]: Okay guys, nvm

Comment (zloyden) [reply filters/filter_20_Annoyances_MobileApp/metadata.json:12]: > sometimes mobile app banners appears in desktop versions of websites and we block them too (or we used to block them)

yep, that's true

Comment (Alex-302) [reply filters/filter_20_Annoyances_MobileApp/metadata.json:12]: There are a lot of the rules for desktop.

File: filters/filter_18_Annoyances_Cookies/metadata.json [hunk 16]
```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 18,
+  "name": "AdGuard Cookies filter",
```
[the remark this answers is not in the document]
Comment (zloyden) [reply filters/filter_18_Annoyances_Cookies/metadata.json]: Changed the current name to the above.

File: locales/en/filters.json (hunk 13 above)
Comment (zloyden) [reply locales/en/filters.json]: Sounds good, I have changed it @el-termikael 

> Cookies

Now it's Cookie Notices

**Comment by DandelionSprout** [comment]
I notice that the various new subscription URLs don't work yet (Leads to 404 pages), which I presume would need to be fixed at some point before this PR goes live:
```
https://filters.adtidy.org/extension/chromium/filters/18.txt
https://filters.adtidy.org/extension/chromium/filters/19.txt
https://filters.adtidy.org/extension/chromium/filters/20.txt
https://filters.adtidy.org/extension/chromium/filters/21.txt
```

It also seems odd to use the Chromium list version as the source for which all other versions would be converted from, instead of the Windows or Mac list versions.

**Review by el-termikael — APPROVED** [review]

**Comment by zloyden** [comment]
Still waiting for translations.

**Review by ss-orlova — APPROVED** [review]

File: locales/ar/filters.json [hunk 17]
```diff
@@ -51,10 +51,6 @@
 		"filter.13.name": "فلتر تركيAdGuard",
 		"filter.13.description": "قائمة التصفية التي تزيل الإعلانات تحديدًا على مواقع الويب باللغة التركية"
 	},
-	{
-		"filter.14.name": "فلتر المضايقاتAdGuard",
-		"filter.14.description": "تحظر العناصر المزعجة على صفحات الويب، بما في ذلك إشعارات ملفات تعريف الارتباط بحاجيات الجهات الخارجية والنوافذ المنبثقة في الصفحة."
-	},
```
Comment (slavaleleka) [inline locales/ar/filters.json:57]: такое не проканает, для **всех наших фильтров** нужны переводы на **все языки**

\+ надо будет обновить в компилере жсон-схему для валидации локалей и потом тут обновить компилер (это на мне)

**Review by Alex-302 — APPROVED** [review]

**Review by aalpatkina — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Review by jellizaveta — APPROVED** [review]

**Review by BlazDT — APPROVED** [review]

**Pull request merged**